### ISSUE TITLE
Wounded Condition

### ISSUE THREAD

**Issue by BrooksUltra** [DFreds/dfreds-convenient-effects#132]
MIDI QOL can apply the "Dead" condition automatically as an overlay.  I have created the custom condition "Wounded", but it only applies as a tiny icon.  It would be great to have Wounded (and perhaps any custom condition) be able to be applied as an overlay. Alternatively, perhaps "Wounded" can be added as a stock condition?

**Comment by DFreds** [comment]
Yes, I actually saw that the other day in Midi. I'll add a stock one. What icon did you decide on for it? Always the hardest part...

**Comment by BrooksUltra** [comment]
> Yes, I actually saw that the other day in Midi. I'll add a stock one. What icon did you decide on for it? Always the hardest part...

Yeah, which icon is the issue.  There is a foundry blood drops icon, but I find it not useful as an overlay since it obscures too much of the token.  I personally like the wounded icon from Combat Utility Belt (Data/modules/combat-utility-belt/icons/wounded.svg).  The condition is obvious with the red tint but the icon does not obscure too much of the token as an overlay.  I would recommend using that as inspiration.

**Comment by DFreds** [comment]
Added in v2.4.4
